target/init.c

```diff
@@ -154,6 +154,11 @@ __WEAK void target_load_ssd_keystore(void)
 {
 }
 
+__WEAK int target_volume_up()
+{
```

Comment:
should not have "cherry picked from" here I think :p

same in some later commits that you re-:cherries:-picked from the previous rebase/tlmm I guess